packages/@aws-cdk/aws-codepipeline-actions/lib/index.ts

```diff
@@ -2,6 +2,9 @@ export * from './alexa-ask/deploy-action';
 export * from './bitbucket/source-action';
 export * from './codestar-connections/source-action';
 export * from './cloudformation/pipeline-actions';
+export * from './cloudformation/stackset-action';
+export * from './cloudformation/stackinstances-action';
+export * from './cloudformation/stackset-types';
```

Comment:
Let's maybe put an `index.ts` in `lib/cloudformation` instead, since we're in the area anyway?